lola.html

```diff
@@ -89,23 +100,31 @@ <h2>Use cases</h2>
-        The primary user need to solve with LOLA is to allow Fediverse users to move their accounts in response to moderation
-        and defederation decisions.  The connections between servers and the communities on servers form a structure that has 
-        close connections, looser connections, and explicit blocks, allowing many different types of communities to co-exist.  
-        That does put a lot of pressure on users to be able to move their accounts (due to moderation decisions they disagree 
-        with, because of server defederation, or just to move closer to an affinity group).   A user making such an account move 
-        will sometimes want to copy their previously posted content to their new location, sometimes follow the same people, and 
-        sometimes notify their followers so their followers can resubscribe in a new location.  Servers implementing LOLA can 
+        One important need to solve with LOLA is to allow Fediverse users to move their accounts in response to moderation
+        and defederation decisions.  The Fediverse allows some close connections (same server), looser connections (federated
+        servers), and explicit blocks (defederated servers), and these connections change over time.
+        That does put a lot of pressure on the system to allow users to move their accounts.  A user moving an account
+        will sometimes want to copy their previously posted content to their new location, sometimes follow the same people, and
+        sometimes notify their followers so their followers can resubscribe in a new location.  Servers implementing LOLA can
         provide these choices to users.
       </p>
 
       <p>
-        Several secondary use cases are enabled by this approach.  It allows accounts to be copied and NOT moved
-        which is useful for testing of all kinds.  The mechanisms are independent of each other (beyond assuming
-        the account migration authorization token), thus allowing reuse.  For instance, the mechanisms for 
-        notify/redirect can
-        also be used after a user moves to a new location using a different approach.
+        Other events may cause users to make very similar decisions.  Users may wish to move to a domain of their own.
+        Fediverse servers can go out of business, servers may change whether/how they charge users, or other decisions
+        may drive the user to a different server.
       </p>
+      <p>One type of use case isn't really a move of content so much as a renaming of URLs (for example, when a service
+        changes domain name or changes its implementation or URL paths such that URLs of Actors and Activities changes).
+        In these use cases, no copying of content is necessary. However, other parts of this specification might be useful
+        in such cases.  The server changing its URLs could use the mechanisms to flag that an item was renamed so that 
+        external references can be fixed over time.  
```

Comment:
```suggestion
      <p>One use case isn't really a movement of content so much as a renaming of URLs (for example, when a service
        changes its domain name or changes its implementation or URL paths such that the URLs of Actors and Activities changes).
        In such use cases, no copying of content is necessary. However, other parts of this specification might be useful
        in such cases; for instance, a server changing its URLs could use the mechanisms to flag that an item was renamed so that 
        external references can be updated over time.  
```